### ISSUE TITLE
SimpleID Usage Qualifcation

### ISSUE THREAD

**Issue by jehunter5811** [blockstack/app-mining#141]
I'm forgoing the template here because this is not an App Mining improvement proposal. It's an independent authentication system built using Blockstack Auth and there needs to be an official determination as to whether applications can use this and be eligible for App Mining. 

Please see Forum thread for more details: 

https://forum.blockstack.org/t/simple-id-easier-blockstack-feature-survey/8476/26 

Expected results from this issue: 

1. Approve or reject SimpleID as an allowable authentication solution in App Mining apps. 

2. If rejected, specify the exact reasons why so that the community may have a voice in the discussion.

**Comment by prabhaav** [comment]
Also, @larrysalibra - can we get your input on if NIL will penalize apps if they add SimpleID as an alternative auth option?

**Comment by friedger** [comment]
See also #100

**Comment by friedger** [comment]
As simple ID is unoptionated maybe some guidelines are needed in order to be eligible.

**Comment by jehunter5811** [comment]
Guidelines for whom? SimpleID or the app developers using it? When we say SimpleID is un-opinionated, what we mean is that we don't think developer tools should force developers down one path or another. Decentralization means choices.

**Comment by friedger** [comment]
I was thinking of guide lines how developers need to present the
authentication. Maybe that is against your un-opinionated vision.

**Comment by prabhaav** [comment]
See tech docs with more details here: https://github.com/simplesecure/ssa-sdk/blob/master/TECHNICAL_DOCS.md

**Comment by jehunter5811** [comment]
@friedger No, I think you're right. If there's a concern that users aren't going to be notified of how this works, Blockstack has every right to ask App Developers to do so in some way. That's the flexibility this affords though. Each developer can communicate however they see fit and still land on the same message.

**Comment by larrysalibra** [comment]
NIL would treat SimpleID the same as any other 3rd party auth system.

**Comment by jeffdomke** [comment]
Agree with @larrysalibra. Moving to review.

**Comment by friedger** [comment]
@jeffdomke The decision here does not match with what @cuevasm said in https://github.com/blockstack/app-mining/issues/148#issuecomment-534395693

Please reopen!
